.github/.goreleaser-unstable.yml

```diff
@@ -0,0 +1,310 @@
+# Copyright (c) Mondoo, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+---
+project_name: cnspec
+env:
+  - CGO_ENABLED=0
+builds:
+  - id: linux
+    main: ./apps/cnspec/cnspec.go
+    binary: cnspec
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - 386
+      - arm64
+      - arm
+      - ppc64le
+    # ARM 6= Raspberry Pi A, A+, B, B+, Zero
+    # ARM 7= Raspberry Pi 2, 3, 4
+    goarm:
+      - 6
+      - 7
+    flags:
+      - -tags="production netgo"
+    ldflags:
+      - "-extldflags=-static"
+      - -s -w -X go.mondoo.com/cnspec.Version={{.Version}} -X go.mondoo.com/cnspec.Build={{.ShortCommit}} -X go.mondoo.com/cnspec.Date={{.Date}}
+      - -X go.mondoo.com/cnquery.Version={{.Version}} -X go.mondoo.com/cnquery.Build={{.ShortCommit}} -X go.mondoo.com/cnquery.Date={{.Date}}
+  - id: macos
+    main: ./apps/cnspec/cnspec.go
+    binary: cnspec
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    flags: -tags production
+    ldflags:
+      # clang + macos does not support static: - -extldflags "-static"
+      - -s -w -X go.mondoo.com/cnspec.Version={{.Version}} -X go.mondoo.com/cnspec.Build={{.ShortCommit}} -X go.mondoo.com/cnspec.Date={{.Date}}
+      - -X go.mondoo.com/cnquery.Version={{.Version}} -X go.mondoo.com/cnquery.Build={{.ShortCommit}} -X go.mondoo.com/cnquery.Date={{.Date}}
+    hooks:
+      post:
+        - cmd: /tmp/quill sign-and-notarize "{{ .Path }}" -vv
+          env:
+            - QUILL_LOG_FILE=/tmp/quill-{{ .Target }}.log
+  - id: windows
+    main: ./apps/cnspec/cnspec.go
+    binary: cnspec
+    goos:
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    # -buildmode exe is required since go 1.15.0 https://github.com/golang/go/issues/40795
+    flags: -tags production -buildmode exe
+    ldflags:
+      - "-extldflags -static"
+      - -s -w -X go.mondoo.com/cnspec.Version={{.Version}} -X go.mondoo.com/cnspec.Build={{.ShortCommit}} -X go.mondoo.com/cnspec.Date={{.Date}}
+      - -X go.mondoo.com/cnquery.Version={{.Version}} -X go.mondoo.com/cnquery.Build={{.ShortCommit}} -X go.mondoo.com/cnquery.Date={{.Date}}
+    hooks:
+      post:
+        - cmd: ./scripts/pkg/windows/windows-upx.sh '{{ .Path }}'
+          output: true
+          env:
+          - TARGET={{ .Target }}
+        - cmd: jsign --storetype DIGICERTONE --alias "{{ .Env.SM_CERT_ALIAS }}" --storepass "{{ .Env.SM_API_KEY }}|{{ .Env.SM_CLIENT_CERT_FILE}}|{{ .Env.SM_CLIENT_CERT_PASSWORD }}" --tsaurl "http://timestamp.digicert.com" '{{ .Path }}'
+nfpms:
```

Comment:
Do we need these? I don't know where they end up.

Reply:
it's an exact copy of the normal goreleaser